src/ConfigManager.cpp

```diff
@@ -358,20 +361,29 @@ void ConfigManager::storeWifiSettings(String ssid, String password, bool resetMa
     char ssidChar[SSID_LENGTH];
     char passwordChar[PASSWORD_LENGTH];
 
+    if (EEPROM.length() == 0) {
+        DebugPrintln(F("WiFi Settings cannot be stored before ConfigManager::begin()"));
+        return;
+    }
+
     strncpy(ssidChar, ssid.c_str(), SSID_LENGTH);
     strncpy(passwordChar, password.c_str(), PASSWORD_LENGTH);
 
-    DebugPrint("Storing WIFI: ");
-    DebugPrintln(ssid);
+    DebugPrint(F("Storing WiFi Settings for SSID: \""));
+    DebugPrint(ssidChar);
+    DebugPrintln(F("\""));
```

Comment:
When the SSID is cleared, it's hard to distinguish between an SSID with whitespace and an empty SSID. This should help with debugging if that's the case.

Reply:
Thanks